### PR TITLE
Adressing petres comments for desktop (responsive hero is still pending)

### PR DESCRIPTION
**Pull request by diemol**
## Home page changes
home_page_changes

## Project page changes
project_page_changes

## Docs page changes
docs_page_changes

### CODE REVIEW

File: static/sass/components/buttons.scss [hunk 1]
```diff
@@ -50,3 +50,22 @@
 .btn-transparent:active, .btn-transparent:hover {
     color: var(--primary-color)
 }
+
+.sauce-button {
+    background: $bgColorSauceButton;
+    background: -webkit-gradient(linear,right top,left top,from($bgColorSauceButtonGradientFrom),to($bgColorSauceButtonGradientTo));
+    background: linear-gradient(270deg,$bgColorSauceButtonGradientFrom,$bgColorSauceButtonGradientTo);
+    box-shadow: -3px 0 16px -11px #000;
+    border-radius: 290486px;
```
Comment (christian-bromann) [inline static/sass/components/buttons.scss]: `290486px` seems like a magic number to me, we can also just say:

```suggestion
    border-radius: 2em;
```

**Review by christian-bromann — COMMENTED** [review]
@diego-saucelabs something is off with the header:

Screenshot 2020-07-06 at 09 34 37

vs.:

Screenshot 2020-07-06 at 09 34 47

**Comment by diemol** [comment]
Yeah, taking the social icons out broke it.
I left them back again and I am now redoing the hero part.

**Comment by diemol** [comment]
@christian-bromann 
So this looks fine again, it is ready to be merged.

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

**Pull request merged**